eclair-core/src/main/scala/fr/acinq/eclair/db/Databases.scala

```diff
@@ -45,6 +46,12 @@ object Databases {
     override val peers = new SqlitePeersDb(eclairJdbc)
     override val payments = new SqlitePaymentsDb(eclairJdbc)
     override val pendingRelay = new SqlitePendingRelayDb(eclairJdbc)
+    override def backup(file: File): Unit = {
```

Comment:
This API makes the assumption that we will only use a single file to store the databases, which is a bit strange.